utils/regr/modamysql.cpp

```diff
@@ -72,17 +75,16 @@ char* moda(CONTAINER& container, struct moda_data* data)
     }
   }
 
-  data->result = std::to_string(val);
-
-  return const_cast<char*>(data->result.c_str());
+  data->result = boost::lexical_cast<std::string>(val);
```

Comment:
Did you consider using std::to_string instead? It's faster and do not use external deps

Reply:
Yes, I had used to_string at first, but the output wasn't as nice as lexical cast for float types. We can switch it back. Float types are a bad choice for this function anyway.

Reply:
Changed it back to to_string